data/pigui/modules/time-window.lua

```diff
@@ -53,24 +47,32 @@ local function displayTimeWindow()
 		ui.sameLine()
 	end
 
-	ui.withFont(pionillium.large.name, pionillium.large.size, function()
-		local text_size = ui.calcTextSize(date)
-		local window_size = Vector2(math.max(text_size.x, (button_size.x + frame_padding * 2 + 7) * 6) + 15, text_size.y + button_size.y + frame_padding * 2 + 22)
+	local text_size
+	ui.withFont(timefont.name, timefont.size, function()
+		text_size = ui.calcTextSize(date)
+	end)
+	local buttons_width = (button_size.x + frame_padding * 2) * 6 + ui.getItemSpacing().x * 5
+		local window_width = math.max(text_size.x, buttons_width) + ui.getWindowPadding().x * 2
+		window_height = timefont.size + button_size.y + frame_padding * 2 + ui.getItemSpacing().y + ui.getWindowPadding().y * 2
+		local window_size = Vector2(window_width, window_height)
 		ui.timeWindowSize = window_size
 		ui.setNextWindowSize(window_size, "Always")
-		ui.setNextWindowPos(Vector2(0, ui.screenHeight - window_size.y), "Always")
-		ui.window("Time", {"NoTitleBar", "NoResize", "NoSavedSettings", "NoFocusOnAppearing", "NoBringToFrontOnFocus"}, function()
+		local window_pos = Vector2(0, ui.screenHeight - window_size.y)
+		ui.setNextWindowPos(window_pos, "Always")
+		ui.window("Time", {"NoTitleBar", "NoResize", "NoSavedSettings", "NoFocusOnAppearing", "NoBringToFrontOnFocus", "NoScrollbar"}, function()
```

Comment:
Indentation mismatch here? Or is this just the github review display messing up again?

Reply:
Guilty, stuck and not corrected.